src/TLuaInterpreter.cpp

```diff
@@ -10879,6 +10884,21 @@ bool TLuaInterpreter::compile(const QString& code, QString& errorMsg, const QStr
     }
 }
 
+// returns true if the given Lua code is valid, false otherwise
+bool TLuaInterpreter::validLuaCode(const QString &code)
```

Comment:
For the record I see this need to be a permanent addition as upstream has closed https://github.com/martin-eden/lua_code_formatter/issues/7 on the basis that it isn't it's job to determine if the code is valid - it requires it to be so.